### ISSUE TITLE
User permissions for editing fields & layout

### ISSUE THREAD

**Issue by mhaarmanalthen** [InvoiceShelf/InvoiceShelf#193]
**Is your feature request related to a problem? Please describe.**
I have a second user who'm I'd like to allow to edit the format on invoices, estimations and payments but even with super admin permissions that user is not allowed to alter those settings. The settings page shown to that user contains fewer tabs. 

**Describe the solution you'd like**
A permission setting that grants all options to the user, like an owner permission level. 

**Describe alternatives you've considered**
My second option would be to spin up a whole new instance of invoiceshelf locally but that would double the resources required.

**Comment by mhaarmanalthen** [comment]
More details:

- I have one instance of invoiceshelf running
- I created two companies, one for myself, one for myself and someone else. 
- In the second company, with my second partner, I would like my partner to have the same rights as I have
- My parter **cannot** edit invoice, estimation and payment field settings such as the invoice number format, expiration date etc. That whole tab is not present for the second user.

**Comment by agencetwogether** [comment]
Any news about this feature ?

**Comment by rihards-simanovics** [comment]
> Any news about this feature ?

No, as of yet. It will be something we look to add; however, when is yet unknown.

**Comment by gdarko** [comment]
Hey @agencetwogether , @rihards-simanovics 

The roles and permission system will be revamped to adjust the application for general use in a way that everyone can register their account and add their company (depending of if that is allowed). Also, in future we will introduce company user invitations which will replace the current "Users" screen.

This task can be categorized as sub-task of or as part of #271

**Comment by gdarko** [comment]
Settings pages like Company Info, Preferences, Customization, and Mail Config are intentionally restricted to company owners. This is by design — owners manage company-wide settings while other users (even super admins of the platform) operate within their assigned role. In v3.0, the role/ability system has been audited and this behavior is consistent. If you need a user to manage settings, they should be the company owner.
